components/are_na/actions/create-channel/create-channel.mjs

```diff
@@ -0,0 +1,35 @@
+import app from "../../are_na.app.mjs";
+
+export default {
+  key: "are_na-create-channel",
+  name: "Create Channel",
+  description: "Create a new channel in Are.na. [See the documentation](https://dev.are.na/documentation/channels#Block45041)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    channelTittle: {
```

Comment:
I think this is supposed to be `channelTitle`?
This is also the case in other components and the app file